### ISSUE TITLE
Volition rules not getting completely deleted

### ISSUE THREAD

**Issue by meldckn** [ensemble-engine/ensemble#86]
(A bug that came up when Michael was showing a test domain during the 11/20/19 EIS meeting.)

Deleting volition rules in the tool doesn't always completely delete them?

## To reproduce:
_As interpreted by me watching Michael trigger this bug only once, without further investigation, because we had to continue the meeting._ 
1. In the tool, load a small domain that at least has a schema category with a type, and volition rule(s) that reference that schema type.
2. Delete all volition rules (that reference the aforementioned schema type) - maybe it's necessary that you delete _all_ volition rules - not sure.
3. Try to delete the schema type (or category?) used in those rules

## The buggy behavior:
The tool will prevent you from deleting the schema type, complaining that a volition rule still exists that references it. Even though, if you click on the volitions tab in the tool, there are none listed. If you open the `volitionRules.json`, at least one (if not more) of the rules that should've been deleted are still there.

## System:
**OS:** macOS (fairly modern version as of 11/20/19) 
**Tool Version:** probably v1.0.1, but otherwise v1.0.0

**Comment by meldckn** [comment]
Closing as duplicate of #67
